FinTech.Logic/BusinessLogic/Account/AccountBL.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FinTech.Data;
+using System;
+
+namespace FinTech.Logic
+{
+    public abstract class AccountBL
+    {
+        public  Account CreateAccount(User user)
+        {
+            Account account = new Account
+            {
```

Comment:
Is there any other ways to new this?

What if I said to new this on multiple places?

Have you repeat the same code on all places?

Reply:
Done